### DISCUSSION TITLE
Is_Dispatching useage

### DISCUSSION THREAD

**Discussion by hughgilhespie** [BottlecapDave/HomeAssistant-OctopusEnergy#1106] [Q&A]
Hi, complete beginner so apologies in advance. Last week our solar PV + battery was installed and have now hit the problem of the house battery discharging into the car. I had read up on this and have tried the various work arounds but they don’t work. So, I am looking for a software solution via HA. I have the myenergi and octopus integrations in HA. I am hoping to set up an automation to set the battery discharge current to zero when Is_Dispatching is true. However, during my last EV charge Is_Dispatching was True for the standard octopus IG period of 23:30 to 05:30. But in the attributes for Is_Dispatching the dispatch periods actually reported the correct schedule as set by Octopus. This was from 00:00 to 06:30. That was the actual charging time. 
I am confused by the different time periods. Would it be possible to set Is_Discharging to the Octopus schedule? If not, is it possible to programmatically extract the schedule?

**Comment by BottlecapDave** [comment]
Hello and sorry for the late response. The dispatching sensor already turns on during off peak or pending dispatch periods. Unfortunately, OE sometimes doesn't report pending dispatch periods and goes straight to completed depending on your integration provider. When this happens, the sensor doesn't know if the sensor should be on.
